server/routes/api/users/users.ts

```diff
@@ -17,155 +17,147 @@ import { rateLimiter } from "@server/middlewares/rateLimiter";
 import { transaction } from "@server/middlewares/transaction";
 import validate from "@server/middlewares/validate";
 import { Event, User, Team } from "@server/models";
-import { UserFlag, UserRole } from "@server/models/User";
+import { UserFlag } from "@server/models/User";
 import { can, authorize } from "@server/policies";
 import { presentUser, presentPolicies } from "@server/presenters";
 import { APIContext } from "@server/types";
 import { RateLimiterStrategy } from "@server/utils/RateLimiter";
 import { safeEqual } from "@server/utils/crypto";
-import {
-  assertIn,
-  assertSort,
-  assertPresent,
-  assertArray,
-} from "@server/validation";
 import pagination from "../middlewares/pagination";
 import * as T from "./schema";
 
 const router = new Router();
 const emailEnabled = !!(env.SMTP_HOST || env.ENVIRONMENT === "development");
 
-router.post("users.list", auth(), pagination(), async (ctx: APIContext) => {
-  let { direction } = ctx.request.body;
-  const { sort = "createdAt", query, filter, ids } = ctx.request.body;
-  if (direction !== "ASC") {
-    direction = "DESC";
-  }
-  assertSort(sort, User);
-
-  if (filter) {
-    assertIn(
+router.post(
+  "users.list",
+  auth(),
+  pagination(),
+  validate(T.UsersListSchema),
+  async (ctx: APIContext<T.UsersListReq>) => {
+    const {
+      sort = "createdAt",
```

Comment:
This looks unnecessary considering that the `schema` already defaults it to `createdAt`?